include/runtime/stackmgr.h

```diff
@@ -100,6 +106,10 @@ class StackManager {
     return Vec;
   }
 
+  Span<Frame> getAllFrames() {
```

Comment:
Since you are visiting all frames without any modification, use `Span<const Frame>` instead.